assets/helpers/types/__tests__/optionTest.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { headOption } from '../option';
+
+
+// ----- Tests ----- //
+
+describe('option', () => {
+
+  describe('headOption', () => {
+
+    it('retrieves the first item from an array', () => {
+      expect(headOption([1, 2, 3])).toBe(1);
+    });
+
+    it('returns null when the array is empty', () => {
+      expect(headOption([])).toBeNull();
+    });
+
+    it('returns null when the array contains null', () => {
```

Comment:
Could we fold this, and the next two tests into `retrieves the first item from an array` above?

Reply:
👍 done.